packages/worker/src/config.ts

```diff
@@ -0,0 +1,13 @@
+import { cleanEnv, EnvError, makeValidator, Spec } from 'envalid'
+
+// custom validator same as str from envalid but with typed return type
+const str = <T extends string = string> (spec?: Spec<T>) => makeValidator((input: string) => {
+  if (typeof input === 'string') return input as T
+  throw new EnvError(`Not a string: "${input}"`)
+})(spec)
+
+type LogLevelString = 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'fatal';
```

Comment:
Is this not available from the `ts-log` package?

Reply:
ah yes it's available from `bunyan` package which is what I'm guessing you meant.

Reply:
I did expect it to be in the ts-log package, but bunyan is fine to source from too.